docs/05.policy/02.modes/02.modes.md

```diff
@@ -38,6 +38,14 @@ In the Network map you can click on any conversation (green, yellow, red line) t
 
 In Protect mode, NeuVector enforcers will block (deny) any network violations and attacks detected. Violations are shown in the Network map with a red ‘x’ in them, meaning they have been blocked. Unauthorized processes and file access will also be blocked in Protect mode. DLP sensors which match will block network connections.
 
+To block traffic in case of network violation, Neuvector enforcers are put inline of network connections, there are two ways of putting enforcer inline.
```

Comment:
```suggestion
In case of any network violation, NeuVector enforcers are put inline of network connections to block the traffic.

There are two ways of putting an enforcer inline:
```